test/k8sT/Updates.go

```diff
@@ -240,12 +239,14 @@ func InstallAndValidateCiliumUpgrades(kubectl *helpers.Kubectl, oldHelmChartVers
 		By("Deploying Cilium %s", oldHelmChartVersion)
 
 		opts := map[string]string{
-			"image.tag":                     oldImageVersion,
-			"operator.image.tag":            oldImageVersion,
-			"hubble.relay.image.tag":        oldImageVersion,
-			"image.repository":              "quay.io/cilium/cilium",
-			"operator.image.repository":     "quay.io/cilium/operator",
-			"hubble.relay.image.repository": "quay.io/cilium/hubble-relay",
+			"image.tag":                              oldImageVersion,
+			"operator.image.tag":                     oldImageVersion,
+			"hubble.relay.image.tag":                 oldImageVersion,
+			"clustermesh.apiserver.image.tag":        oldImageVersion,
+			"image.repository":                       "quay.io/cilium/cilium-ci",
+			"operator.image.repository":              "quay.io/cilium/operator-ci",
+			"hubble.relay.image.repository":          "quay.io/cilium/hubble-relay-ci",
```

Comment:
These options are specific to v1.10. There isn't a v1.9 for `cilium-ci` for example.

Reply:
Why does this matter? This PR is about master (v1.10 -> v1.11).

Reply:
Don't we run nightly jobs that tests the combination of all upgrades? v1.8->v1.9, v1.9-v1.10, etc?

Reply:
Oh, do those run using the master version of the test? I'm not at all familiar with these.

Are those somehow triggered from one of the below paths? I see only policies nigthly tests, not upgrade ones:

```
$ git grep -i Nightly test/
test/Makefile:nightly:
test/Makefile:  ginkgo --focus "Nightly" -v -- --cilium.provision=$(PROVISION) --cilium.registryCredentials=$(REGISTRY_CREDENTIALS)
test/config/config.go:          "Specifies scope of test to be ran (k8s, Nightly, runtime)")
test/helpers/cons.go:// NightlyStableUpgradesFrom maps the cilium image versions to the helm charts
test/helpers/cons.go:// that will be used to run update tests in the Nightly test.
test/helpers/cons.go:var NightlyStableUpgradesFrom = map[string]string{
test/helpers/kubectl.go:        // Sometimes Kubectl returns 126 exit code, It use to happen in Nightly
test/helpers/scope.go:  case strings.Contains(focusString, "nightly"):
test/helpers/scope.go:          // Nightly tests run in a Kubernetes environment.
test/k8sT/PoliciesNightly.go:var _ = Describe("NightlyPolicies", func() {
test/k8sT/StressPolicy.go:var _ = Describe("NightlyPolicyStress", func() {
test/test_suite_test.go:                        // For Nightly test we need to have more than two kubernetes nodes. If
```

Who can I ask / how can I find out more about this?

Reply:
> Don't we run nightly jobs that tests the combination of all upgrades? v1.8->v1.9, v1.9-v1.10, etc?

I can't find any such thing in https://jenkins.cilium.io. We simply run this same K8sUpdates test regularly on master as part of the usual scheduled CI runs.

Reply:
I'd imagine we regularly run each upgrade as part of the scheduled CI runs for each branch, but that would use the upgrade tests code from each individual branch to upgrade from the prior release.

@aanm if you have more details, I can look into what other changes might be required but for now I don't see how this can break other jobs.

Reply:
The code changes look good to me as it is. However I'll ping @nebril and he will have the definitive answer to what happened to the Nightly tests.